src/Microsoft.Azure.SignalR.Common/Constants.cs

```diff
@@ -10,8 +10,10 @@ internal static class Constants
         public const string ServerStickyModeDefaultKey = "Azure:SignalR:ServerStickyMode";
         public const string ConnectionStringDefaultKey = "Azure:SignalR:ConnectionString";
         public const string ApplicationNameDefaultKey = "Azure:SignalR:ApplicationName";
+        public const string EnableAutoScaleDefaultKey = "Azure:SignalR:EnableAutoScale";
```

Comment:
`"Azure:SignalR:AutoScale": true`?

Reply:
Enable by default and reverted the change.